python/dgl/convert.py

```diff
@@ -701,7 +649,17 @@ def to_hetero(G, ntypes, etypes, ntype_field=NTYPE, etype_field=ETYPE,
 
     return hg
 
-def to_homo(G):
+def to_hetero(G, ntypes, etypes, ntype_field=NTYPE, etype_field=ETYPE,
+              metagraph=None):
+    """Convert the given homogeneous graph to a heterogeneous graph.
+
+    DEPRECATED: Please use to_heterogeneous
+    """
+    dgl_warning("dgl.to_hetero is deprecated. Please use dgl.to_heterogeneous")
+    return to_heterogeneous(G, ntypes, etypes, ntype_field=ntype_field,
+                            etype_field=etype_field, metagraph=metagraph)
+
+def to_homogeneous(G):
```

Comment:
Can we fix this?
https://github.com/dmlc/dgl/pull/1721

Reply:
Done. 1) `ntypes`, `etypes`, and `canonical_etypes` are always sorted. 2) We will remove node/edge features about type/ID with `to_heterogeneous`.